src/elements/pages/dev/game-banner.ts

```diff
@@ -0,0 +1,36 @@
+import { LitElement, TemplateResult, html } from 'lit';
+import { customElement, property } from 'lit/decorators.js';
+import styles from './game-banner.scss';
```

Comment:
We can remove the SCSS file completely if it's empty. Iirc `cssify` supports passing no params.